src/Evolution/DiscontinuousGalerkin/InterfaceDataPolicy.hpp

```diff
@@ -0,0 +1,72 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstdint>
+#include <iosfwd>
+
+namespace evolution::dg {
+/// \brief Label for a neighboring Element (or Block) that determines how
+/// information is exchanged between neighboring Elements
+///
+/// \details The specific label is determined by the relationship between the
+/// block logical coordinates of the neighboring Elements (Blocks) for the
+/// points on the interface between them.  In two cases there is a simple
+/// relationship between the coordinates:
+/// - CopyProject: in this case the block logical coordinates are identical.
+///   Therefore a DataVector can be either copied (if the Mesh on each side of
+///   the interface are at the same points) or projected to the Mesh of the
+///   neighbor.
+/// - OrientCopyProject: in this case the block logical coordinates are related
+///   by a discrete rotation (represented by an OrientationMap). Therefore a
+///   DataVector can be reoriented (with the OrientationMap), and then either
+///   copied or projected to the Mesh of the neighbor.
+/// In the following cases, there is no simple relationship between the block
```

Comment:
Doxygen wants a blank line after the bulleted list for formatting.  Again below.